docs/content/en/docs/implementing/tasks-non-k8s-apps.md

```diff
@@ -4,23 +4,24 @@ description: Using Keptn with Non-Kubernetes Applications
 weight: 95
 ---
 
-Keptn Tasks can be triggered for workloads and applications
+Keptn Tasks running on a Kubernetes cluster
+can be triggered for workloads and applications
 that are deployed outside of Kubernetes.
 For example, Keptn could trigger load and performance tests
 for an application that is deployed on a virtual machine.
 
 To do this:
 
-1. [Install and enable a Kubernetes cluster](#install-and-enable-a-kubernetes-cluster)
+1. [Install Keptn on a Kubernetes cluster](#install-keptn-on-a-kubernetes-cluster)
 1. [Create a KeptnTaskDefinition](#create-a-keptntaskdefinition)
 1. [Create and apply a KeptnTask](#create-and-apply-a-keptntask)
 1. [Re-run the KeptnTask](#re-run-the-keptntask)
 
-## Install and enable a Kubernetes cluster
+## Install Keptn on a Kubernetes cluster
 
-You must still
+You must enable a Kubernetes cluster and
```

Comment:
the wording "enable a Kubernetes cluster" doesn't click well on me :/ 

maybe "You must [install](../install/install.md/#use-helm-chart) Keptn on a Kubernetes cluster"

Reply:
i would go for "set up"

Reply:
I went with Mo's wording, so it "You must set up a Kubernetes cluster and install Keptn on it"